src/til/ut_til/RunLengthEncodingTests.cpp

```diff
@@ -361,15 +361,10 @@ class RunLengthEncodingTests
 
             rle.replace(test_case.start_index, test_case.end_index, change);
 
-            try
-            {
-                VERIFY_ARE_EQUAL(test_case.expected, rle);
-            }
-            catch (...)
-            {
-                // I couldn't figure out how to attach additional info
-                // to a failed assertion so I'm doing it this way...
-                Log::Comment(NoThrowString().Format(

```

Comment:
I think there's another instance of "I couldn't figure out..." in this file :smile: (line 420-430?)